frame/benchmarking/Cargo.toml

```diff
@@ -17,6 +17,8 @@ sp-std = { version = "2.0.0-alpha.2", path = "../../primitives/std", default-fea
 sp-io = { path = "../../primitives/io", default-features = false, version = "2.0.0-alpha.2" }
 frame-support = { version = "2.0.0-alpha.2", default-features = false, path = "../support" }
 frame-system = { version = "2.0.0-alpha.2", default-features = false, path = "../system" }
+pallet-collective = { version = "2.0.0-alpha.2", default-features = false, path = "../collective" }
+node-primitives = { version = "2.0.0-alpha.2", default-features = false, path = "../../bin/node/primitives" }
```

Comment:
I don't think having a pallet and a node crate as dependency of frame-benchmarking is suitable this is contradictory to their seperation IMO.